File: nodejs/database/models/hospital.js
'use strict';

module.exports = (sequelize, DataTypes) => {
    const hospital = sequelize.define('hospital', {
        id_hospital: {
            type: DataTypes.INTEGER,
            primaryKey: true,
            autoIncrement: true
        },
        name: {
            type: DataTypes.STRING(50),
            defaultValue: null
        }
    }, {
        freezeTableName: true,
        underscored: true,
        name: {
            singular: 'hospital',
            plural: 'hospital'
        }
    });

    hospital.associate = models => {
        hospital.hasMany(models.doctor, {
            foreignKey: 'id_hospital',
            onDelete: 'cascade'
        });
    };

    return hospital;
};

File: python/src/models/hospital.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship
from database import Base


class Hospital(Base):
    __tablename__ = "hospital"
    id_hospital = Column(Integer, primary_key=True, nullable=False, autoincrement=True)
    name = Column(String(50), nullable=False)
    doctors = relationship("Doctor", back_populates="hospital")
